cognee/modules/data/methods/__init__.py

```
@@ -7,6 +7,7 @@
 from .get_datasets_by_name import get_datasets_by_name
 from .get_dataset_data import get_dataset_data
 from .get_data import get_data
+from .get_unique_dataset_id import get_unique_dataset_id
```

Ruff (0.8.2)

`.get_unique_dataset_id.get_unique_dataset_id` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)